xclim/indices/_conversion.py

```diff
@@ -2017,3 +2019,157 @@ def mean_radiant_temperature(
+        out = wind_speed * (h / h_r) ** alpha
+        out.attrs["units"] = wind_speed.attrs["units"]
+        return out
+    else:
+        raise NotImplementedError(f"Method {method} not implemented.")
+
+
+@declare_units(
+    wind_speed="[speed]",
+    air_density="[air_density]",
+    cut_in="[speed]",
+    rated="[speed]",
+    cut_out="[speed]",
+)
+def wind_power_potential(
+    wind_speed: xr.DataArray,
+    air_density: xr.DataArray = None,
+    cut_in: Quantified = "3.5 m/s",
+    rated: Quantified = "13 m/s",
+    cut_out: Quantified = "25 m/s",
+) -> xr.DataArray:
+    r"""Wind power potential estimated from a semi-idealized wind power production factor.
+
+    The actual power production of a wind farm can be estimated by multiplying its nominal (nameplate) capacity by the
+    wind power potential, which depends on wind speed at the hub height, the turbine specifications and air density.
+
+    Parameters
+    ----------
+    wind_speed : xarray.DataArray
+        Wind speed at the hub height. Use the `wind_profile` function to estimate from the surface wind speed. Note
+        that the temporal resolution of wind time series has a significant influence on the results. Mean daily wind
+        speeds yield lower values than hourly wind speeds.
+    air_density: xarray.DataArray
+        Air density at the hub height. Defaults to 1.225 kg/m³. This is worth changing if applying in cold or
+        mountainous regions with non-standard air density.
+    cut_in : Quantified
+        Cut-in wind speed. Default is 3.5 m/s.
+    rated : Quantified
+        Rated wind speed. Default is 13 m/s.
+    cut_out : Quantified
+        Cut-out wind speed. Default is 25 m/s.
+
+    Returns
+    -------
+    xr.DataArray
+      The power production factor. Multiply by the nominal capacity to get the actual power production.
+
+    See Also
+    --------
+    wind_profile : Estimate wind speed at the hub height from the surface wind speed.
+
+    Notes
+    -----
+    This estimate of wind power production is based on a semi-idealized power curve with four wind regimes specified
+    by the cut-in wind speed (:math:`u_i`), the rated speed (:math:`u_r`) and the cut-out speed (:math:`u_o`).
+    Power production is zero for wind speeds below the cut-in speed, increases cubically between the cut-in
+    and rated speed, is constant between the rated and cut-out speed, and is zero for wind speeds above the cut-out
+    speed to avoid damage to the turbine :cite:p:`tobin_2018`:
+
+    .. math::
+
+        \begin{cases}
+        0,  &  v < u_i \\
+        (v - u_i)^3 / (u_r - u_i)^3,  & u_i ≤ v < u_r \\
+        1, & u_r ≤ v < u_o \\
+        0, & v ≥ u_o
+        \end{cases}
+
+    For non-standard air density (:math:`\rho`), the wind speed is scaled using
+    :math:`v_n = v \left( \frac{\rho}{\rho_0} \right)^{1/3}`.
+
+    Note that the temporal resolution of the wind speed time series has a significant influence on the results,
```

Comment:
Perhaps this is not necessary, but you could also note that temporal resolution may affect aggregation in time

Reply:
Changed to 

Note that the temporal resolution of the wind speed time series has a significant influence on the results,
    even when aggregated at longer time scales. Total annual power production will differ substantially if estimated
    from hourly or daily wind speeds, due to the non-linearity of the production factor.

Is this what you meant ?